--- bluefish-2.2.12/desktop-data/bluefish.appdata.xml.in-orig	2022-02-05 19:06:20.566005585 -0500
+++ bluefish-2.2.12/desktop-data/bluefish.appdata.xml.in	2022-02-05 19:06:41.048185783 -0500
@@ -26,9 +26,9 @@
 <launchable type="desktop-id">bluefish.desktop</launchable>
 
 <categories>
-​    <category>Development</category>
-​    <category>Webdevelopment</category>
-​</categories>
+    <category>Development</category>
+    <category>Webdevelopment</category>
+</categories>
 
 <screenshots>
  <screenshot type="default">
@@ -41,4 +41,10 @@
 <kudo>HiDpiIcon</kudo>
 <kudo>ModernToolkit</kudo>
 </kudos>
+<content_rating type="oars-1.0" />
+<releases>
+	<release version="2.2.12" date="2020-11-05" />
+	<release version="2.2.11" date="2017-01-27" />
+	<release version="2.2.10" date="2020-01-24" />
+</releases>
 </component>
